plugins/maciVoting/services/coordinator/config.ts

```diff
@@ -0,0 +1,34 @@
+import { EPolicies, EMode, EInitialVoiceCreditProxies } from "@maci-protocol/sdk";
+import { type IDeployPollConfig } from "./utils/types";
+import { Keypair, PrivateKey } from "@maci-protocol/domainobjs";
+
+const MSG_BATCH_SIZE = 20;
+
+const startDate = Math.floor(Date.now() / 1000) + 100;
+const pollDuration = 60;
+
+// TODO: what should I do with this?
+const coordinatorMACIKeypair = new Keypair(
+  PrivateKey.deserialize("macisk.bdd73f1757f75261a0c9997def6cd47519cad2856347cdc6fd30718999576860")
```

Comment:
Why is it hardcoded?

Reply:
I wasn't sure how auth was meant to work here. It is now removed based on Nico's suggestions